prac_05/state_names.py

```diff
@@ -0,0 +1,28 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+STATE_NAMES = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+               "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+
+# print(STATE_NAMES)
```

Comment:
leverage this location to function call the printing of the states, you will need to add a main() etc.